packages/design-tokens/webpack.config.js

```diff
@@ -1,5 +1,3 @@
 module.exports = {
-	entry: {
-		TalendDesignTokens: ['./src/index.ts', './src/index.scss'],
-	},
+	entry: './src/index.umd.ts',
```

Comment:
Why not use the previous version with the entry array?

Reply:
* What could it means ?
* UMD is supposed to reprensent one js file with externals... not a simple script, so what could we export.
* current umd config is one entry point: https://unpkg.com/@talend/scripts-config-react-webpack@12.0.0/config/webpack.config.umd.js
* webpack doc specify it: https://webpack.js.org/configuration/entry-context/#entry
> https://webpack.js.org/configuration/entry-context/#entry

Looking at the current result if you try to use the produced umd you have

```javascript
> require('@talend/design-tokens/dist/TalendDesignTokens').default
undefined
> require('@talend/design-tokens/dist/TalendDesignTokens')
Object [Module] {}
```

So this multiple entrypoint (array or object) do not make sens with UMD.

Reply:
I feel it weird to get that `index.umd` into that `src/` folder but ok
Maybe it's the current index which makes no sense actually, let's see that in another PR